exampleCourse/serverFilesCourse/python_autograder/run.sh

```diff
@@ -45,12 +43,17 @@ sudo chmod -R +rw $BIN_DIR
 
 echo "[run] starting autograder"
 
-# we do the capturing ourselves, so that only the stdout of the autograder is used and that we aren't relying on any files that the student code could easily create
-# we are also running the autograder as a limited user called ag
-sudo -H -u ag bash -c 'python3.5 autograder_wrapper.py' > results.json
+# we run the autograder as a limited user called ag
+bash -c 'python3.5 pltest.py'
```

Comment:
`python3` to make it easier to upgrade the grading container in the future?

Reply:
There was a reason we used `python3.5` in the first place, though I can't quite remember why :)

Reply:
Hmm, `python3` seems to work fine. I've made the change.